### ISSUE TITLE
[Feature Request]: Mathematical Built-in function exp()

### ISSUE THREAD

**Issue by dengn** [matrixorigin/matrixone#1822]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Describe the solution you'd like

Returns the value of e (the base of natural logarithms) raised to the power of X. 
```
mysql> SELECT EXP(2);
        -> 7.3890560989307
mysql> SELECT EXP(-2);
        -> 0.13533528323661
mysql> SELECT EXP(0);
        -> 1
```

### Documentation, Adoption, Use Case, Migration Strategy

Please refer to the [tutorial](https://docs.matrixorigin.io/0.3.0/MatrixOne/Contribution-Guide/Tutorial/develop_builtin_functions/) about how to develop a builtin function.

**Comment by Y7n05h** [comment]
Please close this issue. This issue has been resolved in #1891 .

**Comment by dengn** [comment]
Thanks for your contribution. @Y7n05h
